helpers/actaRecibido/actaRecibidoCrud.go

```diff
@@ -32,3 +31,26 @@ func GetElementoById(id int) (elemento *models.Elemento, outputError map[string]
 
 	return elemento, nil
 }
+
+func GetAllElemento(query string) (elementos []*models.Elemento, outputError map[string]interface{}) {
```

Comment:
Falta documentar, cambiar `...` en el _suggested change_, digamos "GetAllElemento hace tal cosa"
```suggestion
// GetAllElemento ...
func GetAllElemento(query string) (elementos []*models.Elemento, outputError map[string]interface{}) {
```